zebrad/src/components/inbound/tests/fake_peer_set.rs

```diff
@@ -308,7 +316,7 @@ async fn mempool_advertise_transaction_ids() -> Result<(), crate::BoxError> {
     Ok(())
 }
 
-#[tokio::test]
+#[tokio::test(flavor = "multi_thread")]
 async fn mempool_transaction_expiration() -> Result<(), crate::BoxError> {
```

Comment:
This test also needs the same fix for the second request:
>  Message:  internal error: entered unreachable code: `MempoolTransactionIds` requests should always respond `Ok(Vec<UnminedTxId>)`
> Location: zebrad/src/components/inbound/tests/fake_peer_set.rs:487

https://github.com/ZcashFoundation/zebra/runs/7423391225?check_suite_focus=true#step:3:6828

It's a bit further down and hidden in the diff:
https://github.com/ZcashFoundation/zebra/pull/4199/files#diff-6a7425f2d6ab0ef475e13cb6412322b72023e7139a23e04f0bc70205846d42b6R482-R490

Reply:
I fixed this by updating the existing test fix commit.